test/classes/diten/multipledestructor.chpl

```diff
@@ -1,7 +1,7 @@
 class C {
-  proc ~C() { writeln("In ~C #1"); }
+  proc deinit() { writeln("In deinit #1"); }
 }
-proc C.~C() { writeln("In ~C #2"); }
+proc C.deinit() { writeln("In deinit #2"); }
```

Comment:
Note: if we take Mike's approach and merge this PR now, I would still prefer this change occur today.  I can also do it myself, if you don't want to but don't otherwise object